### PR TITLE
Setup ghc api

### PR DESCRIPTION
**Pull request by ralren**
## Description

Hey good folks, this does all the necessary set up to use GHC in the back and front end.

## Reviewer Notes

Should the API be false in environments outside of local?

## Setup
Make sure http://officelocal:8080/ghc/v1/swagger.yaml and http://officelocal:3000/ghc/v1/swagger.yaml show the yaml files.

```sh
make server_run
make office_client_run
```

### CODE REVIEW

**Comment by chrisgilmerproj** [comment]
I won't be able to see this today but definitely look at https://github.com/transcom/mymove/pull/2788 for hints on stuff I changed (especially the documentation) for the TSP stuff. But at first glance this looks great.

File: pkg/cli/services.go [hunk 1]
```diff
@@ -37,13 +40,15 @@ func CheckServices(v *viper.Viper) error {
...
+	ghcAPIEnabled := v.GetBool(ServeGHCFlag)
 
 	// Oops none of the flags used
 	if (!adminEnabled) &&
 		(!sddcEnabled) &&
 		(!ordersEnabled) &&
 		(!dpsEnabled) &&
-		(!internalAPIEnabled) {
+		(!internalAPIEnabled) &&
+		(!ghcAPIEnabled) {
```
Comment (chrisgilmerproj) [inline pkg/cli/services.go]: If mutual TLS is enabled then either Orders or GHC should be enabled: https://github.com/transcom/mymove/pull/2790/files#diff-de0700612bb446c72954d65be742717aR55-R62

Comment (ralren) [reply pkg/cli/services.go]: @chrisgilmerproj when you have more time can we hop on a zoom about this. I'm not sure if I understand your meaning.

Comment (chrisgilmerproj) [reply pkg/cli/services.go]: I think maybe I'm confused. This must be the Task Order API and not the PRIME API. If it was the PRIME API then we'd want to include the Mutual TLS check. So you're probably good.

**Pull request merged**